docs/developer/DEVELOP.md

```diff
@@ -82,12 +82,8 @@ TRACKERS_SYNC_TO_JIRA=1
 
 # Run taskman tests behind a proxy (optional)
 # This variable is only necessary if rewriting the taskman cassettes locally and using the Stage Red Hat JIRA instance, which requires a proxy to be accessed.
-# Some tests apply the HTTPS_PROXY variable hardcoded in respective conftest files.
 HTTPS_TASKMAN_PROXY="http://foo.bar"
```

Comment:
The naming is a bit confusing now as the first sentence inclines it is only for taskman tests but second sentence also mentions Jira. Maybe we should really just name `HTTPS_JIRA_PROXY` so it feels more general? I don't have a strong preference but somehow this feels weird :smile: